### DISCUSSION TITLE
Creating new reference image results in device reset

### DISCUSSION THREAD

**Discussion by jasonkaisersmith** [jomjol/AI-on-the-edge-device#3173] [General]
For some reason my image has suddenly gone out of alignment and I am getting a negative flow error, as it can no longer read the values.

Trying to correct this, when trying to create a new reference image the UI freezes and it appears the device reboots.  So it is not possible to create a new reference.  I have tried this several times.

When entering the config page to set debug log level, I received many errors that several parameters are incorrect (mostly relating to zoom and AEC control).  I corrected these by setting the default values in the raw edit mode (it was not possible via UI).

Attached are the log file, config zip, reference image, and last image.
[error_20040513_jason_kaisersmith.zip](https://github.com/jomjol/AI-on-the-edge-device-docs/files/15292748/error_20040513_jason_kaisersmith.zip)

**Comment by cthulu** [comment]
Same issue here, I've fixed the config but taking a new reference crashes.

**Comment by marcelk1101** [comment]
Same issue for me. Cant take a new image. Had to get back to 15.6.0

Will it be fixed in the next Release?

**Comment by schrej-zz** [comment]
Same issue for me. No response yet?
EDIT: ISSUE solved by using another SD-Card as recommended.

**Comment by caco3** [comment]
If it suddenly starts to crash then most of the times it is because the SD-Card went bad.
Sometimes it also is just enough to re-format the SD-card. Make sure to backup all files in the `config` folder. The only other folder which is needed is the `html` folder which contains the Web UI.
